src/Evolution/EventsAndDenseTriggers/DenseTriggers/Or.hpp

```diff
@@ -0,0 +1,136 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <algorithm>
+#include <limits>
+#include <memory>
+#include <pup.h>
+#include <pup_stl.h>
+#include <utility>
+#include <vector>
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "Evolution/EventsAndDenseTriggers/DenseTrigger.hpp"
+#include "Options/Options.hpp"
+#include "Options/ParseOptions.hpp"
+#include "Parallel/CharmPupable.hpp"
+#include "Time/EvolutionOrdering.hpp"
+#include "Time/TimeStepId.hpp"
+#include "Utilities/Algorithm.hpp"
+#include "Utilities/Registration.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+namespace Tags {
+struct TimeStepId;
+}  // namespace Tags
+/// \endcond
+
+namespace DenseTriggers {
+template <typename TriggerRegistrars>
+class Or;
+
+namespace Registrars {
+using Or = Registration::Registrar<DenseTriggers::Or>;
+}  // namespace Registrars
+
+/// \ingroup EventsAndTriggersGroup
+/// Trigger when any of a collection of triggers triggers.
```

Comment:
maybe make the second last `triggers` -> `DenseTriggers` or `Triggers` to make it a bit easier to parse?